enterprise/com/src/test/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpackerTest.java

```diff
@@ -374,21 +418,41 @@ public void shouldNotMarkTransactionsAsCommittedIfAppenderClosed() throws Throwa
     private TransactionAppender mockedTransactionAppender() throws IOException
     {
         TransactionAppender appender = mock( TransactionAppender.class );
-        when( appender.append( any( TransactionRepresentation.class ), anyLong() ) ).thenReturn( mock( Commitment.class ) );
+        when( appender.append( any( TransactionRepresentation.class ), anyLong() ) )
+                .thenReturn( new StubbedCommitment() );
         return appender;
     }
 
-    private void setUpIndexUpdatesValidatorMocking( DependencyResolver dependencyResolverMock ) throws IOException
+    private static class StubbedCommitment implements Commitment
```

Comment:
I was just reluctant to add a test dependency from `ha` to `com`. I'll move `StubbedCommitment` to kernel (src/test) and both `com` and `ha` can enjoy it since they already depend on kernel tests.